src/Augurk/Controllers/V2/FeatureV2Controller.cs

```diff
@@ -98,24 +95,14 @@ public async Task<DisplayableFeature> GetFeatureAsync(string productName, string
         /// <returns>Returns a reponse message indicating whether saving the feature succeeded.</returns>
         [Route("{title}/versions/{version}")]
         [HttpPost]
-        public async Task<HttpResponseMessage> PostAsync(Feature feature, string productName, string groupName, string title, string version)
+        public async Task<ActionResult<Feature>> PostAsync([FromBody]Feature feature, string productName, string groupName, string title, string version)
```

Comment:
The [FromBody[ here is now required with ASP.NET Core.

Reply:
I actually feel that to be cleaner. I like having that one being explicit.

Reply:
I agree, it just tripped me up because we didn't have these in place and it only failed at runtime. I think I got all of the instances where we needed this though ;).